tox.ini

```diff
@@ -32,6 +34,25 @@ commands = python -m pytest \
            --junitxml {toxworkdir}/junit.{envname}.xml \
            {posargs:.}
 
+
+[testenv:mypy]
+description = check that the type annotations within attrs-strict are self-consistent. Does NOT check if stub files align correctly with actual code.
+basepython = python3.7
+deps = mypy
+setenv =
+    {[testenv]setenv}
+    MYPYPATH={toxinidir}
+commands = mypy types --py2 --strict
+           mypy types --strict
+
+[testenv:merge]
+description = try to merge our types against our source
```

Comment:
I don't see a reason for merge and mypy to not be in the same type tox target environment and merge the files in a tempdir 🤷